lib/helpers/results_helpers.js

```diff
@@ -139,7 +127,12 @@ module.exports = class ResultsHelpers {
       statementRelevance[libraryName][statementName] === 'FALSE'
     ) {
       statementRelevance[libraryName][statementName] = relevant ? 'TRUE' : 'FALSE';
-      return Array.from(cqlStatementDependencies[libraryName][statementName]).map(dependentStatement =>
+      const library = cqlStatementDependencies.find(lib => lib.library_name === libraryName);
+      const statement = library.statement_dependencies.find(stat => stat.statement_name === statementName);
+      if (!statement.statement_references) {
```

Comment:
In the off chance `statement` is `undefined`, can we add a check to make sure it exists before we attempt to access `statement_references`?